lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -672,7 +672,8 @@ void TypeAnalysis::visitOperation(Operation *op,
           AtenTriuOp, AtenMaskedFillTensorOp, AtenRollOp, AtenPowTensorTensorOp,
           AtenLiftFreshCopyOp, AtenIndexTensorHackedTwinOp,
           AtenUpsampleNearest2dOp, AtenMishOp, AtenRoundOp, AtenFillTensorOp,
-          AtenUpsampleNearest2dBackwardOp, AtenLeakyReluBackwardOp>(op)) {
+          AtenUpsampleNearest2dBackwardOp, AtenLeakyReluBackwardOp,
+          AtenOneHotOp>(op)) {
```

Comment:
This types of changes should be accompanied by full e2e support and testing, otherwise there is no way of knowing if the code here is working the same as PyTorch. See this discussion: https://github.com/llvm/torch-mlir/pull/1799#discussion_r1091045107